tests/test_queries.py

```diff
@@ -0,0 +1,890 @@
+            project=ProjectConstraintsModel(
+                project_id="dsgrid_conus_2022",
+                include_dsgrid_dataset_components=True,
+                dataset_ids=[
+                    "conus_2022_reference_comstock",
+                    "conus_2022_reference_resstock",
+                    # "tempo_conus_2022",
+                ],
+                # TODO: need to allow min/max/sum of metric on other supplemental dimensions
+                dimension_filters=[
+                    DimensionFilterExpressionModel(
+                        dimension_type=DimensionType.GEOGRAPHY,
+                        query_name="county",
+                        operator="==",
+                        value="06037",
+                    ),
+                ],
+                metric_reductions=[
+                    MetricReductionModel(
+                        dimension_query_name="electricity",
+                        operation=self._operation,
+                    ),
+                ],
+                # filtered_datasets=[],
+            ),
+        )
+        return self._model
+
+    def get_output_electricity_op(self, operation):
+        df = self._spark.read.parquet(str(self.output_dir / self.name / "table.parquet"))
+        value = perform_op(df, f"all_electricity_{operation.__name__}", operation)
+        return value
+
+    def validate(self):
+        county = self.get_filtered_county_id()
+        expected_base = self.get_raw_stats()["by_county"][county]["comstock_resstock"]
+        if self._operation == "sum":
+            expected = expected_base["sum"]["electricity"]
+            actual = self.get_output_electricity_op(F.sum)
+        elif self._operation == "max":
+            expected = expected_base["max"]["electricity"]
+            actual = self.get_output_electricity_op(F.max)
+        else:
+            logger.warning("Validation is not yet supported for %s", self._operation)
+            return True
+
+        result = math.isclose(expected, actual)
+        if not result:
+            ratio = actual / expected
+            logger.error(
+                "Mismatch in values. actual=%s expected=%s ratio=%s", actual, expected, ratio
+            )
+        return result
+
+
+class QueryTestElectricityValues(QueryTestBase):
+
+    NAME = "electricity-values"
+
+    def make_query(self):
+        self._model = ProjectQueryResultModel(
+            name=self.NAME,
+            project=ProjectConstraintsModel(
+                project_id="dsgrid_conus_2022",
+                include_dsgrid_dataset_components=True,
+                dataset_ids=[
+                    "conus_2022_reference_comstock",
+                    "conus_2022_reference_resstock",
+                    # "tempo_conus_2022",
+                ],
+                dimension_filters=[
+                    DimensionFilterColumnOperatorModel(
+                        dimension_type=DimensionType.METRIC,
+                        query_name="electricity",
```

Comment:
Where does this come from? How does this differ from `dimension_query_name` = electricity?